inst/Upgrades.qml

```diff
@@ -7,7 +7,55 @@ Upgrades
 	{
 		functionName: 		"SEMSimple"
 		newFunctionName:	"SEM"
-		fromVersion:		"0.14.2"
+		fromVersion:		"0.14.3"
```

Comment:
I thought the previous version was 0.14.1?

Reply:
There were some additional for linux see: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-upgrade-qml.md#migrating-from-oldschool-modules

Reply:
Ah, okay!